tensorflow/python/ops/structured/structured_tensor_test.py

```diff
@@ -900,14 +900,25 @@ def testTupleFieldValue(self):
     st = StructuredTensor.from_pyval({"a": 5, "b": {"c": [1, 2, 3]}})
     self.assertAllEqual(st.field_value(("a",)), 5)
     self.assertAllEqual(st.field_value(("b", "c")), [1, 2, 3])
-    with self.assertRaisesRegexp(KeyError,
-                                 r"Field path \('a', 'b'\) not found in .*"):
+    expected = "Field path \(.*a.*,.*b.*\) not found in .*"
```

Comment:
Adding the field values in `__repr__` caused extra slashes to escape the quotes, and apparently they end up in the exception string. To simplify the expected string, I used any characters around the field names in place of the slashes and quotes.